hack/ci-e2e.sh

```diff
@@ -70,7 +70,15 @@ virsh -c qemu:///system attach-interface --domain minikube --mac="52:54:00:6c:3c
 
 # Restart minikube to apply the changes
 minikube stop
-minikube start
+## Following loop is to avoid minikube restart issue
+## https://github.com/kubernetes/minikube/issues/14456
+while /bin/true; do
+    minikube_error=0
+    minikube start || minikube_error=1
```

Comment:
This is simpler with just 
```suggestion
    if minikube start; then break; fi
```
Also adding a sleep 5 or sleep 30 (whatever is appropriate) is good, instead of just spamming. Some exit condition in case minikube doesn't recover would also be nice, until we deem quitting after timeout (45mins or so) is OK.

Reply:
Or even

```suggestion
    while ! minikube start; do sleep 30; done
```

Reply:
agree, this is simpler and less spammed, done